docroot/themes/humsci/humsci_airy/humsci_airy.info.yml

```diff
@@ -0,0 +1,18 @@
+name: HumSci Airy
+type: theme
+base theme: humsci_basic
+description: A light and airy theme.
+package: Humsci
+core: 8.x
+core_version_requirement: ^8 || ^9
+
+regions:
```

Comment:
these are copied from Su_humsci theme right? just trying to remember where we were copying from

Reply:
Right.